base/setup/lib/setuplib.c

```diff
@@ -628,18 +628,24 @@ InitDestinationPaths(
     IN PPARTENTRY PartEntry)    // FIXME: HACK!
 {
     WCHAR PathBuffer[MAX_PATH];
-
-    //
-    // TODO: Check return status values of the functions!
-    //
+    NTSTATUS Status;
 
     /* Create 'pSetupData->DestinationRootPath' string */
     RtlFreeUnicodeString(&pSetupData->DestinationRootPath);
-    RtlStringCchPrintfW(PathBuffer, ARRAYSIZE(PathBuffer),
-            L"\\Device\\Harddisk%lu\\Partition%lu\\",
-            DiskEntry->DiskNumber,
-            PartEntry->PartitionNumber);
-    RtlCreateUnicodeString(&pSetupData->DestinationRootPath, PathBuffer);
+    Status = RtlStringCchPrintfW(PathBuffer, ARRAYSIZE(PathBuffer),
+                     L"\\Device\\Harddisk%lu\\Partition%lu\\",
+                     DiskEntry->DiskNumber,
+                     PartEntry->PartitionNumber);
+
+    /* Check the condition status of RtlStringCchPrintfW() */
+    if (!NT_SUCCESS(Status))
+    {
+        DPRINT("RtlStringCchPrintfW() failed with status 0x%08lx\n", Status);
```

Comment:
This doesn't require a change but for future reference, the `08` is just noise here. Status is by definition larger than 0x80000000, so it will always print with 8 digits anyway.